container/bin/entrypoint.py

```diff
@@ -0,0 +1,75 @@
+#!/venv/bin/python3
+#
+# SPDX-FileCopyrightText: Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+import argparse
+import json
+import os
+import subprocess
+
+from ovirt_imageio.client import info
```

Comment:
Do we really need client.info? This is a more fancy implementation that supports 
getting info from a file inside tar file, but here we want to do a basic 
qemu_img.info() call, so we should really import qemu_img instead.

Also importing functions from modules is bad, making it harder to understand the
code. Where is info coming from? Now we need to scroll up to check, and we may
shadow this name by a local variable.